### PR TITLE
Added UpdateExpandedTextAd example

### PR DESCRIPTION
**Pull request by fiboknacky**

### CODE REVIEW

File: examples/BasicOperations/UpdateExpandedTextAd.php [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        // change.
+        $adOperation = new AdOperation();
+        $adOperation->setUpdate($ad);
+        $adOperation->setUpdateMask(FieldMasks::allSetFieldsOf($ad));
+
+        // Issues a mutate request to update the ad.
+        $adServiceClient = $googleAdsClient->getAdServiceClient();
+        $response = $adServiceClient->mutateAds($customerId, [$adOperation]);
+
+        // Prints the resource name of the updated ad group criterion.
```
Comment (PierrickVoulet) [inline examples/BasicOperations/UpdateExpandedTextAd.php]: The comment is about ad group criterion instead of ad.

**Review by PierrickVoulet — CHANGES_REQUESTED** [review]

File: examples/BasicOperations/UpdateExpandedTextAd.php [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+ */
+
+namespace Google\Ads\GoogleAds\Examples\BasicOperations;
+
+require __DIR__ . '/../../vendor/autoload.php';
+
+use GetOpt\GetOpt;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentNames;
+use Google\Ads\GoogleAds\Examples\Utils\ArgumentParser;
+use Google\Ads\GoogleAds\Lib\OAuth2TokenBuilder;
```
Comment (wfansh) [inline examples/BasicOperations/UpdateExpandedTextAd.php]: Nit: I saw in other samples (UpdateAdGroup.php, UpdateCampaign.php) the OAuth2TokenBuilder import is right above FieldMasks. Shall we update here for consistency?

File: examples/BasicOperations/UpdateExpandedTextAd.php [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $adId
+    ) {
+        // Creates an ad with the specified resource name and other changes.
+        $ad = new Ad([
+            'resource_name' => ResourceNames::forAd($customerId, $adId),
+            'expanded_text_ad' => new ExpandedTextAdInfo([
+                'headline_part1' => new StringValue(['value' => 'Cruise to Pluto #' . uniqid()]),
+                'headline_part2' => new StringValue(['value' => 'Tickets on sale now'])
```
Comment (wfansh) [inline examples/BasicOperations/UpdateExpandedTextAd.php]: 'description' attribute might be missed:
Golden CL: https://github.com/googleads/google-ads-dotnet/blob/master/examples/BasicOperations/UpdateExpandedTextAd.cs#L81

**Review by wfansh — CHANGES_REQUESTED** [review]

File: examples/BasicOperations/UpdateExpandedTextAd.php (hunk 3 above)
Comment (fiboknacky) [reply examples/BasicOperations/UpdateExpandedTextAd.php]: Nice catch. I followed Python. Just pinged Fei to fix this too.

File: examples/BasicOperations/UpdateExpandedTextAd.php (hunk 2 above)
Comment (fiboknacky) [reply examples/BasicOperations/UpdateExpandedTextAd.php]: Those examples are pretty old and chances are I didn't sort them properly.
This one should be correct.

Comment (wfansh) [reply examples/BasicOperations/UpdateExpandedTextAd.php]: Noted.

**Review by wfansh — APPROVED** [review]

**Review by PierrickVoulet — APPROVED** [review]

**Pull request merged**